frontend/src/components/CreateCollectionModal/components/CTA/index.tsx

```diff
@@ -27,7 +28,7 @@ const CTA: FC<Props> = ({ onClose }) => {
           </Button>
           <AnchorButton
             intent={Intent.PRIMARY}
-            href={`${API_URL}${API.LOG_IN}`}
+            href={`${API_URL}${API.LOG_IN}?redirect=?${QUERY_PARAMETERS.LOGIN_MODULE_REDIRECT}`}
```

Comment:
I haven't checked out the code yet 😆 But I see a question mark after `redirect=`, is that intended? 

Feels like that'd interpolate to `?redirect=?showCCModule` instead of `?redirect=showCCModule`

Thank you!!

Reply:
I'm 85% sure that the backend requires the query `?` as it would also accept something along the lines of `route?param`

Reply:
And I think we might need `?redirect=showCCModule=true`, so the redirect URL will be `?showCCModule=true`, which is a valid URL param?

Reply:
Ahh sorry my bad, you're right! We do need two question marks 😆

And then likely `=true` too?

Reply:
I'm just doing a check to check that the query param is present and not set to false, lmk if you prefer it more verbose

Reply:
Yeah if you don't mind, let's make it consistent with other URL param implementations, so we follow the "convention over customization" idea that Rails framework popularized!

Also, checking the query param exists and not set to false sounds like the deny list approach over the preferred allow list approach, which potentially means a typo in param chaining `showCCModule=foo=true` could trigger the modal too!

What do you think 😄 ? Thank you!

Reply:
Consistency is key.  Sounds good to me!

Interesting thoughts about deny/allow list approaches and that edge case. I took a look and you're right about failing open.

`http://localhost:8000/?showCCModule=auth=true`
results in
![image](https://user-images.githubusercontent.com/8716829/101393436-f8b9ef80-387b-11eb-9025-528f9d20a40d.png)

Reply:
@seve Ah I think we forgot to add `=true` here 😆 

`href={`${API_URL}${API.LOG_IN}?redirect=?${QUERY_PARAMETERS.LOGIN_MODULE_REDIRECT}`=true}`

PTAL! Thank you!